activerecord/lib/active_record/relation/query_methods.rb

```diff
@@ -78,6 +78,18 @@ def #{method_name}=(value)           # def includes_values=(value)
 
     alias extensions extending_values
 
+    def load_associations(arr, *args)
```

Comment:
`array` is better than `arr` but I think we should also support relations.

Reply:
Could be `loaded_records_or_relation`.